gui/elements/qt/_imageLayer.py

```diff
@@ -0,0 +1,52 @@
+from PyQt5.QtWidgets import QSlider, QComboBox, QHBoxLayout, QGroupBox, QVBoxLayout, QCheckBox
+from PyQt5.QtCore import Qt
+
+class QtImageLayer(QGroupBox):
```

Comment:
I think that since all layers are expected to have `opacity` and `visible` properties, this aspect should be abstracted away.

Instead, we "sandwich" the customization in between the visibility and opacity components.

Another option could be just to create a base class which would result in better modularity.

@Napari/gui-design What do you think?

Reply:
I like the idea of having a base class. As you mentioned, having a base class is more modular. Additionally, I think it would make it more clear to contributors how to create new layers.

Reply:
Yes, this all makes sense. I'll give it a go with a base class and then some customization. I think it's also important for us to think about what sort of interactivity we want for each of the layer types. My thought right now is just to get something sensible in there, and then we can make future PRs that refine the widgets - i.e. adding control over marker `symbol` as well etc.